engine/src/main/scala/cromwell/engine/io/RetryableRequestSupport.scala

```diff
@@ -51,25 +51,30 @@ object RetryableRequestSupport {
   /**
     * ATTENTION: Transient failures are retried *forever*
     * Be careful when adding error codes to this method.
-    * Currently only 429 (= quota exceeded are considered truly transient)
     */
-  def isTransient(failure: Throwable): Boolean = failure match {
-    case gcs: StorageException => gcs.getCode == 429
-    case _ => false
+  def isInfinitelyRetryable(failure: Throwable): Boolean = {
+
+    // Retry forever because eventually the user will have quota capacity again
```

Comment:
I have a feeling that "out of quota" is sometime used to indicate "you don't have any of that resource in this region". I guess it's ok to keep trying infinitely since we already were... but I'm nevertheless a little nervous about this!

Reply:
According to the docs [0], `429` responses from GCS should be met with "truncated exponential backoff" [1].

I wasn't sure what the "truncated" part meant, and Wikipedia helpfully explains [2]:

> The 'truncated' simply means that after a certain number of increases, the exponentiation stops; i.e. the retransmission timeout reaches a ceiling, and thereafter does not increase any further.

They don't mention an absolute limit on time to retry or number of attempts, which implicitly leads me to believe that infinite is OK.

[0] https://cloud.google.com/storage/docs/json_api/v1/status-codes#429_Too_Many_Requests
[1] https://cloud.google.com/storage/docs/exponential-backoff
[2] https://en.wikipedia.org/wiki/Exponential_backoff